//
// Created by User on 20/10/2022.
//
#include "stdexcept"

static void Test(){

    throw std::runtime_error("error");

}